server/routes/appointments/create/handlers/location.ts

```diff
@@ -32,8 +32,7 @@ export default class LocationRoutes {
       .then(locations => locations.find(l => l.locationId === locationId))
 
     if (!location) {
-      req.flash('validationErrors', JSON.stringify([{ field: 'locationId', message: `Selected location not found` }]))
-      return res.redirect('back')
+      return res.validationFailed('locationId', `Selected location not found`)
     }
```

Comment:
Will this change affect all validation, appointments or activities? Or is this interchangeable and a choice to use it or not?

Reply:
This was a change suggested by Stephen. The validationFailed extension method reduces code duplication but it's definitely optional whether you call it or not. It currently is only used for this custom, in handler validation logic. The existing `validationMiddleware` that all standard validation uses has not been changed.